tests/GeneratedHydratorTestAsset/ClassWithPrivatePropertiesAndParent.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/*
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ * This software consists of voluntary contributions made by many individuals
+ * and is licensed under the MIT license.
+ */
+
+declare(strict_types=1);
+
+namespace GeneratedHydratorTestAsset;
+
+/**
+ * Base test class to play around with parent class private properties
+ *
+ * @author Marco Pivetta <ocramius@gmail.com>
+ * @author Pierre Rineau <pierre.rineau@makina-corpus.com>
+ * @license MIT
+ */
+class ClassWithPrivatePropertiesAndParent extends ClassWithPrivateProperties
+{
+    private $property0 = 'property0_fromChild';
+
+    private $property1 = 'property1_fromChild';
+
+    private $property20 = 'property20';
+
+    protected $property21 = 'property21';
+
+    public $property22 = 'property22';
+
+    /**
+     * @param string $property0
+     */
+    public function setProperty0($property0)
```

Comment:
This was written from a copy past of another test class, which doesn't carry any return type. It's definitely not needed. I'm OK to write them, but for consistency all test classes should probably reviewed and fixed too. Not sure this is the scope of the issue, once again I'd like to have @Ocramius' opinion about this.